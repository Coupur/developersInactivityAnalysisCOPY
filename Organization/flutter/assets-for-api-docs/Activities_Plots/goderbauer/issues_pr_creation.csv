id,date,creator_login
388762052,2018-12-07 18:18:40,goderbauer
388764536,2018-12-07 18:26:13,goderbauer
399098850,2019-01-14 22:20:02,goderbauer
