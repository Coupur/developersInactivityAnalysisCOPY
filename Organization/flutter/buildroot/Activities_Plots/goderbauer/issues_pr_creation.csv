id,date,creator_login
202280261,2017-01-21 01:27:34,goderbauer
202597900,2017-01-23 17:24:49,goderbauer
202606233,2017-01-23 17:58:04,goderbauer
216244932,2017-03-23 00:05:53,goderbauer
239324412,2017-06-28 23:16:57,goderbauer
341145471,2018-07-13 20:11:42,goderbauer
341605860,2018-07-16 17:21:44,goderbauer
341718735,2018-07-16 23:58:01,goderbauer
344584943,2018-07-25 19:57:16,goderbauer
355804127,2018-08-31 01:01:31,goderbauer
356087822,2018-08-31 18:47:45,goderbauer
356090205,2018-08-31 18:56:02,goderbauer
357305272,2018-09-05 16:19:49,goderbauer
357381056,2018-09-05 20:00:51,goderbauer
357822024,2018-09-06 20:49:23,goderbauer
358576990,2018-09-10 11:14:24,goderbauer
358955487,2018-09-11 09:11:21,goderbauer
359057359,2018-09-11 13:51:14,goderbauer
359109912,2018-09-11 15:50:48,goderbauer
359432945,2018-09-12 11:15:11,goderbauer
359854267,2018-09-13 11:26:51,goderbauer
359966608,2018-09-13 16:15:38,goderbauer
368004454,2018-10-09 01:11:15,goderbauer
369813776,2018-10-13 15:43:04,goderbauer
371901591,2018-10-19 10:36:34,goderbauer
375132549,2018-10-29 17:30:50,goderbauer
375197260,2018-10-29 20:16:31,goderbauer
376921791,2018-11-02 18:29:26,goderbauer
381804335,2018-11-17 00:16:53,goderbauer
382807368,2018-11-20 19:07:19,goderbauer
382889443,2018-11-20 23:23:37,goderbauer
382917778,2018-11-21 01:33:03,goderbauer
389505830,2018-12-10 22:02:01,goderbauer
390069639,2018-12-12 05:22:47,goderbauer
390078919,2018-12-12 06:10:31,goderbauer
