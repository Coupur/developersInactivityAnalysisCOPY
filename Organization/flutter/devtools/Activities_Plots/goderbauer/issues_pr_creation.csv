id,date,creator_login
454392875,2019-06-10 22:02:00,goderbauer
