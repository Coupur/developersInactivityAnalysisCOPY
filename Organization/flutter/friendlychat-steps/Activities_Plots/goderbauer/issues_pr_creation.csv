id,date,creator_login
242140866,2017-07-11 18:25:54,goderbauer
242212316,2017-07-11 23:20:52,goderbauer
