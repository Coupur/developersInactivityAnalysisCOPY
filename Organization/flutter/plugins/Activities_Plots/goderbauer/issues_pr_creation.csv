id,date,creator_login
227734821,2017-05-10 16:16:16,goderbauer
227743365,2017-05-10 16:46:12,goderbauer
227746885,2017-05-10 16:59:28,goderbauer
228068497,2017-05-11 17:39:13,goderbauer
228073489,2017-05-11 17:58:48,goderbauer
228081357,2017-05-11 18:29:33,goderbauer
228087022,2017-05-11 18:51:28,goderbauer
228088389,2017-05-11 18:57:07,goderbauer
228376717,2017-05-12 18:47:25,goderbauer
228380184,2017-05-12 19:01:47,goderbauer
228391913,2017-05-12 19:54:45,goderbauer
228396020,2017-05-12 20:11:53,goderbauer
228402207,2017-05-12 20:38:53,goderbauer
228433389,2017-05-12 23:56:16,goderbauer
228873562,2017-05-15 23:23:19,goderbauer
228879938,2017-05-16 00:06:35,goderbauer
229121212,2017-05-16 18:07:13,goderbauer
229162762,2017-05-16 20:42:19,goderbauer
229194130,2017-05-16 23:10:10,goderbauer
229194511,2017-05-16 23:12:37,goderbauer
229196897,2017-05-16 23:27:59,goderbauer
231120180,2017-05-24 17:48:36,goderbauer
231132751,2017-05-24 18:35:51,goderbauer
233656171,2017-06-05 17:27:30,goderbauer
234664997,2017-06-08 21:43:58,goderbauer
234677552,2017-06-08 22:50:19,goderbauer
234681632,2017-06-08 23:15:41,goderbauer
236539253,2017-06-16 16:58:55,goderbauer
236542584,2017-06-16 17:14:28,goderbauer
237038635,2017-06-19 22:10:18,goderbauer
237051859,2017-06-19 23:29:39,goderbauer
237290933,2017-06-20 17:43:11,goderbauer
237689113,2017-06-21 23:07:00,goderbauer
241026280,2017-07-06 17:13:47,goderbauer
243119102,2017-07-14 21:39:20,goderbauer
243463550,2017-07-17 17:08:34,goderbauer
245140821,2017-07-24 16:45:15,goderbauer
260369072,2017-09-25 18:18:46,goderbauer
319252463,2018-05-01 16:40:58,goderbauer
319310845,2018-05-01 20:09:53,goderbauer
319326668,2018-05-01 21:01:10,goderbauer
319367672,2018-05-01 23:59:36,goderbauer
328347898,2018-06-01 01:16:50,goderbauer
350960547,2018-08-15 20:45:23,goderbauer
418548405,2019-03-07 22:42:07,goderbauer
