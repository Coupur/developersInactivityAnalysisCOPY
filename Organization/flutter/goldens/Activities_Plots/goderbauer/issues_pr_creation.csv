id,date,creator_login
338050579,2018-07-03 20:54:43,goderbauer
358965100,2018-09-11 09:36:19,goderbauer
359433620,2018-09-12 11:17:19,goderbauer
359441376,2018-09-12 11:41:41,goderbauer
377608363,2018-11-05 22:22:45,goderbauer
393597731,2018-12-21 21:37:58,goderbauer
